framework/src/executioners/Eigenvalue.C

```diff
@@ -51,6 +51,11 @@ Eigenvalue::validParams()
   params.addParam<Real>(
       "normal_factor", 1.0, "Normalize eigenvector to make a defined norm equal to this factor");
 
+  params.addParam<bool>("auto_initialization",
```

Comment:
This parameter is not like the one in the old eigenvalue executioner. The old eigenvalue executioner does not have slepc. The default value of this parameter is true. When this parameter is set to true, MOOSE will initialize the eigenvector to const 1. And this initialization is done in Executioner init. When `execute` is called again, initialization will not be done again.

Reply:
We do not want to do `initialization` in the second `execute` because you want to reuse the solution from the previous step in Picard iteration.